public/paysagerenderer.js

```diff
@@ -89,12 +85,16 @@ function createLayer (targetCanvas, code) {
   // The compilation step is split from the creation of the
   // Processing object so that we can hook the onLoad event
   // to set width and height correctly before setup() runs.
+  try {
   var sketch = window.Processing.compile(code);
   sketch.onLoad = resize;
   var newLayer = new window.Processing(targetCanvas, sketch);
   setDefaultBackgroundToTransparent(newLayer);
   patchBackgroundFunctionToBeTransparentByDefault(newLayer);
   return newLayer;
+  } catch (e) {
+      console.error('Error in code object ' + targetCanvas + '. Code not rendered. ' + e);
```

Comment:
Apart from the broken indents that ESLint caught, I really was thinking of `updateObject` rather than `createLayer` for the `catch`. Among other things, you don’t have the object ID in `createLayer`, so the error message is bound to be ugly.